app/app.go

```diff
@@ -1075,6 +1076,8 @@ func (app *Evmos) setupUpgradeHandlers() {
 		// no store upgrades in v2
 	case v4.UpgradeName:
 		// no store upgrades in v4
+	case v5.UpgradeName:
+		// no store upgrades in v5
```

Comment:
@ramacarlucho How come the claims records migration is not considered a store upgrade?

Reply:
This is just to indicate that no modules were added